plugins/scaffolder/src/types.ts

```diff
@@ -64,3 +64,16 @@ export type ListActionsResponse = Array<{
     output?: JSONSchema;
   };
 }>;
+
+type OutputLink = {
+  url: string;
+  title?: string;
+  icon?: string;
+};
+
+export type TaskOutput = {
+  entityRef?: string;
+  remoteUrl?: string;
```

Comment:
Should totally mark `remoteUrl` deprecated as part of this, but keep `entityRef` since that has some needed logic.